src/database-sqlite3.cpp

```diff
@@ -56,6 +65,50 @@ SQLite format specification:
 			 sqlite3_errmsg(m_database)); \
 	}
 
+static int sqlite3BusyHandler(void *data, int count)
```

Comment:
The `sqlite3` prefix is for the `sqlite3` library.  Just use `(private) static int Database_SQLite3::busyHandler(...)`.